src/index.js

```diff
@@ -118,14 +119,25 @@ class ReactTooltip extends Component {
     let targetArray = this.getTargetArray()
 
     let dataEvent
+    let dataEventOff
     for (let i = 0; i < targetArray.length; i++) {
       if (targetArray[i].getAttribute('currentItem') === null) {
         targetArray[i].setAttribute('currentItem', 'false')
       }
       dataEvent = this.state.event || targetArray[i].getAttribute('data-event')
       if (dataEvent) {
-        targetArray[i].removeEventListener(dataEvent, this.checkStatus)
-        targetArray[i].addEventListener(dataEvent, this.checkStatus, false)
+        dataEventOff = this.state.eventOff || targetArray[i].getAttribute('data-event-off');
+        // if off event is specified, we will show tip on data-event and hide it on data-event-off
+        if (dataEventOff) {
+          targetArray[i].removeEventListener(dataEvent, this.showTooltip);
```

Comment:
Yeah, pretty much, but checkStatus also creates a listener for `when click other place, the tooltip should be removed`, so I don't know if dataEventOff can compatible with this feature or maybe we don't need that feature if user has set dataEventOff. I need to think about it and advices are welcome.